stores/gamestop.js

```diff
@@ -0,0 +1,85 @@
+import axios from "axios";
+import moment from "moment";
+import DomParser from "dom-parser";     // https://www.npmjs.com/package/dom-parser
+import open from "open"
+import { fileURLToPath } from "url";

```

Comment:
You forgot this :P

Reply:
Definitely did thanks for catching it. 😀